### PR TITLE
Add - split discount line into variable and fixed fee

### LINKED ISSUE #2628: Split the discount line further into two parts (Variable fee and fixed fee)
From: https://github.com/Automattic/woocommerce-payments/pull/2500#issuecomment-889330454
Split the discount line in transaction timeline further into two parts:

### Fee breakup

image

### PR DESCRIPTION
**Pull request by naman03malhotra**
Fixes #2628 

#### Changes proposed in this Pull Request
This PR further splits the discount line into two for a better UX. More context: https://github.com/Automattic/woocommerce-payments/pull/2500#issuecomment-889330454

#### Testing instructions

- Make sure you have a discount enabled. You can do so by running `wp wcpay add_fee_structure_to_account <id> --fee_structure_id=promo-1`
- Make a transaction preferably which includes int. and fx fee.
- Make sure you see fee breakdown that includes discount line.

image

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

### CODE REVIEW

**Review by htdat — CHANGES_REQUESTED** [review]
Looks good to me in general. I have left an inline comment and a following question: 

- Is that necessary to make changes on `package-lock.json`? IMHO, if we'd like to make changes for this file, it should be in a separate issue?

File: client/payment-details/timeline/map-events.js [hunk 1]
```diff
@@ -277,20 +277,21 @@ const feeBreakdown = ( event ) => {
...
 	} );
 
+	const renderDiscountSplit = (
+		percentageRateFormatted,
+		fixedRateFormatted
+	) => {
+		return (
+			<ul className="discount-split-list">
+				<li>Variable fee: { percentageRateFormatted }%</li>
+				<li>Fixed fee: { fixedRateFormatted }</li>
```
[the remark this answers is not in the document]
Comment (naman03malhotra) [reply client/payment-details/timeline/map-events.js]: Ah! right, addressed here: https://github.com/Automattic/woocommerce-payments/pull/2684/commits/81931637a91be31d2b28431b37145008dbf382e3

**Comment by naman03malhotra** [comment]
>Is that necessary to make changes on package-lock.json

I did a fresh install for all the node modules, you are right, we shouldn't include this, so I removed it.

**Comment by naman03malhotra** [comment]
Hey @htdat 👋

Thanks for the first round of review, this is ready for the second round of review.

**Review by htdat — APPROVED** [review]
LGTM 🚢

**Pull request merged**